blueocean-executor-info-plugin/README.md

```diff
@@ -0,0 +1,25 @@
+# Blue Ocean Executor Info Plugin
```

Comment:
This file too seems out of date now, and safe to delete.

Reply:
Done